.github/workflows/presubmit.yml

```diff
@@ -3,6 +3,7 @@ on: [push, pull_request]
 
 jobs:
   build:
+    if: false
```

Comment:
I'm not familiar with GitHub actions.  Does this disable this entire build?

Reply:
Yes, but you're right, I should delete the entire file. The rework is a replacement to `presubmit.yml`.